package.json

```diff
@@ -53,7 +53,7 @@
         "local-storage": "^2.0.0",
         "lodash": "^4.17.21",
         "near-fastauth-wallet": "^0.0.8",
-        "near-social-vm": "github:NearSocial/VM#2.4.2",
+        "near-social-vm": "github:NearSocial/VM#2.5.0",
```

Comment:
Hey @roshaans. Please pull the lates version of `develop` branch because we already use `2.5.1` version. I think you did it before we made an update of that